docs/news/posts/2024-12-13.md

```diff
@@ -0,0 +1,107 @@
+---
+blogpost: true
+category: Blog
+tags: plugins
+author: Julian Geiger
+date: 2024-12-13
+---
+
+# Non research-domain specific plugins to extend core functionality
+
+Dear users, we're happy to announce that we have recently crossed the 100-plugin mark of registered plugin packages on the [AiiDA plugin registry](https://aiidateam.github.io/aiida-registry/).
+Among those are classics like [`aiida-quantumespresso`](https://github.com/aiidateam/aiida-quantumespresso/) and [`aiida-vasp`](https://github.com/aiidateam/aiida-vasp/), to drive the electronic structure codes we all know and love.
+However, did you know that there are various non research-domain specific plugins that AiiDA developers have created over the years, which can extend AiiDA's core functionality?
+
+In today's post, we would therefore like to give you a short introduction to each of these plugins, and show you when you can incorporate them into your work, to make sure you'll get the most out of your AiiDA journey![^1]
+
+## Simplifying code execution and workflow creation
+
+### [`aiida-workgraph`](https://github.com/aiidateam/aiida-workgraph/)
+
+We've been making quite a bit of fuzz about this one in recent weeks already, as we're targeting a first, stable release for early next year (fingers crossed :crossed_fingers:).
+The `aiida-workgraph` provides a new, alternative way of construction workflows, in addition to the classical `WorkChain`.
+It comes with all the typical AiiDA features, like provenance tracking, checkpointing, caching, and remote execution, but provides a simpler syntax to construct workflows.
+This can be achieved by adding nodes (or `Tasks`, as we call them) and links, in equivalence to how one would typically build-up a graph.
```

Comment:
Should we not immediately avoid the "node" nomenclature here?

Reply:
Yeah, good point. Probably it's confusing with AiiDA's `orm.Node` otherwise. Maybe we can specify "graph nodes" and add a footnote not to confuse them with `orm.Node`s? I would not avoid the word, and just use `Task`, though, otherwise people won't know what we're talking about, I think ^^